app/models/queue_item.rb

```diff
@@ -0,0 +1,26 @@
+class QueueItem < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :video
+
+  delegate :category, to: :video
+  delegate :title, to: :video, prefix: :video
+
+  before_create :set_order
+
+  default_scope { order(:order) }
```

Comment:
Be very careful with `default_scope`, it can have unintended consequences throughout your app. It is usually better to use scopes.